src/components/Login/index.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import { Form, Input, Button, Checkbox, Alert } from "antd";
+import { useHistory } from "react-router-dom";
+import firebase from "../../firebaseConfig";
+import { MailOutlined } from "@ant-design/icons";
+import { Link } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+
+const layout = {
+  labelCol: { span: 4 },
+  wrapperCol: { span: 8 },
+};
+
+const tailLayout = {
+  wrapperCol: { offset: 8, span: 16 },
+};
+
+const Login = ({ isTesting }) => {
+  const { t } = useTranslation();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [successMessage, setSuccessMessage] = useState("");
+
+  const signin = (e) => {
+    e.preventDefault();
+    firebase
+      .auth()
+      .signInWithEmailAndPassword(email, password)
+      .then(() => {
+        setSuccessMessage(true);
+      })
+      .catch((err) => {});
+  };
+
+  let history = useHistory();
+  const routeChange = () => {
+    let path = "/forgot";
+    history.push(path);
+  };
+
+  const resetPassword = (e) => {
+    routeChange();
+  };
+
+  return (
+    <div className="formInputSection" {...layout} {...tailLayout}>
+      <Form>
+        {successMessage ? (
```

Comment:
I think for the seccess message you can do it with one if condition so please delete line 57